src/cli_args/mod.rs

```diff
@@ -40,7 +40,7 @@ pub struct Cli {
 
     #[arg(
         long,
-        help = "Absolute path to save esp_idf.json file. Default is $HOME/.esp_installation_manager/esp_idf.json"
+        help = "Absolute path to save eim_idf.json file. Default is $HOME/.esp_installation_manager/eim_idf.json"
```

Comment:
Can we update this help output to list the default path for eim_idf.json in Windows and POSIX ?

Reply:
done